Microsoft.Toolkit.Uwp.UI.Controls/HeaderedTextBlock/HeaderedTextBlock.cs

```diff
@@ -75,5 +79,28 @@ private void UpdateForOrientation(Orientation orientationValue)
                     break;
             }
         }
+
+        private void UpdateInlines()
+        {
+            if (_textContent == null || Inlines.Count == 0)
+            {
+                return;
+            }
+
+            _textContent.Inlines.Clear();
+
+            foreach (var inline in Inlines)
+            {
+                _textContent.Inlines.Add(inline);
+            }
+
+            // To follow the behavior of UWP use Text instead of Inlines if Text property is set.
+            if (!string.IsNullOrEmpty(Text))
+            {
+                var temp = Text;
```

Comment:
Why is this needed?

Reply:
Without this following XAML-sample are different; and I do not want to have different behaviors:

~~~
<StackPanel Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
    <controls:HeaderedTextBlock
        Header="@[Header:String:Name]"
        Text="@[Text:String:UWP Toolkit]"
        Orientation="@[Orientation:Enum:Orientation.Vertical]"
        Margin="20,10,0,0">
        <controls:HeaderedTextBlock.Inlines>
            <Bold>
                <Run Text="@[Header:String:Name]" />
            </Bold>
            <Italic>
                <Run Text="@[Header:String:Name]" />
            </Italic>
        </controls:HeaderedTextBlock.Inlines>
    </controls:HeaderedTextBlock>
    <TextBlock Text="@[Text:String:UWP Toolkit]">
        <TextBlock.Inlines>
            <Bold>
                <Run Text="@[Header:String:Name]" />
            </Bold>
            <Italic>
                <Run Text="@[Header:String:Name]" />
            </Italic>
        </TextBlock.Inlines>
    </TextBlock>
</StackPanel>
~~~

Without this code the `Inlines` are shown in `HeaderedTextBlock`; but according to `TextBlock` the `Text` should